solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -469,7 +477,7 @@ public boolean isClosed() {
       if (cc != null) cc.securityNodeChanged();
     });
 
-    init(registerOnReconnect);
+    init();
```

Comment:
Just looking in GitHub review here but did you omit this param because it's already in scope?

Reply:
It was unused in the method.